src/SeaNetMicron.cpp

```diff
@@ -53,35 +53,44 @@ void Micron::decodeSonarBeam(base::samples::SonarBeam &sonar_beam)
     }
 
     //copy data into SonarBeam
-    sonar_beam.time = base::Time::now();
-    sonar_beam.sampling_interval  = ((640.0*data.ad_interval)*1e-9);
-    sonar_beam.bearing = base::Angle::fromRad(-(data.bearing/6399.0*2.0*M_PI)+M_PI);
```

Comment:
Here it is the same issue. If you don't intend to change the frame of the sonar measurements you will end up with an inverted image. This angle here expresses a rotation around the z-axis.

> So if the bearing decreases, the legacy sonar viewer (SonarView) will plot the data from left to right.

That sounds right.